docs/markdown/snippets/relative_path_within_native_and_crossfiles.md

```diff
@@ -0,0 +1,5 @@
+## Relative paths are supported within native and cross files
+
+Support for relative paths within native and cross files has been added. 
+This reduces the initial effort when projects include the toolchain 
+within the repository.
```

Comment:
It should specify relative to what? Users might assume it's relative to the location of the file instead of relative to top source dir.

Reply:
Also should specify that it's not assumed to be relative in the case it's just a filename with no path. I guess we could suggest `./foo` in that case.

Reply:
Should be documented in `Machine-files.md` too, snippets/ is for release notes only. You can just copy/paste the text.

Reply:
I'll work your comments in.